setup.py

```diff
@@ -201,6 +201,7 @@
 
 extras_require.update({
     ':python_version == "3.4"': ['typing'],
+    ':python_version >= "3.5"': ['trio', 'curio'],
```

Comment:
Neither of these are dependencies, right? We shouldn't require these packages.

Reply:
One of them is necessary if we want to have  nested `embed()`.  I can make that optional, I probably added them here to have the test passing.